src/component.js

```diff
@@ -65,17 +64,19 @@ const MinimumViableComponent = function(element) {
 
 MinimumViableComponent.prototype = Object.assign(
   {
-    // a nice symmetry to [TODO: React Lifecycle Methods] exists here.
+    // A nice symmetry to [React Lifecycle Methods][] exists here.
     // these are the required methods to implement. You may additionally provide
     // custom implementations of other lifecycle methods or any arbitrary
     // methods that are private to your implementation.
+    //
+    // [React Lifecycle Methods]: https://facebook.github.io/react/docs/component-specs.html#lifecycle-methods
```

Comment:
I decided to use [reference-style link syntax](https://daringfireball.net/projects/markdown/syntax#link) here because I think it reads more easily as plaintext, but it's up to you!